[INFO] [2021-06-15 21:31:54,947] [2573:139793912252224] - task_executor.py[line:313]: report task 2021061521315031191914_dataio_0 0 host 9998 to driver
[INFO] [2021-06-15 21:31:55,403] [2573:139793912252224] - task_executor.py[line:150]: Run 2021061521315031191914 dataio_0 2021061521315031191914_dataio_0 host 9998 task
[INFO] [2021-06-15 21:31:55,403] [2573:139793912252224] - task_executor.py[line:151]: Component parameters on party {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': False, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense', 'need_run': True}, 'initiator': {'role': 'guest', 'party_id': 9999}, 'dsl_path': './examples/LDPDev/test_predict_dsl.json', 'job_parameters': {'job_type': 'predict', 'work_mode': 0, 'backend': 0, 'computing_engine': 'STANDALONE', 'federation_engine': 'STANDALONE', 'storage_engine': 'STANDALONE', 'engines_address': {'computing': {'nodes': 1, 'cores_per_node': 20}, 'federation': {'nodes': 1, 'cores_per_node': 20}, 'storage': {'nodes': 1, 'cores_per_node': 20}}, 'federated_mode': 'SINGLE', 'task_parallelism': 1, 'computing_partitions': 4, 'federated_status_collect_type': 'PULL', 'model_id': 'guest-9999#host-9998#model', 'model_version': '202106152108460902496', 'eggroll_run': {'eggroll.session.processors.per.node': 4}, 'spark_run': {}, 'rabbitmq_run': {}, 'pulsar_run': {}, 'adaptation_parameters': {'task_memory_per_node': 0, 'if_initiator_baseline': False, 'task_cores_per_node': 4, 'request_task_cores': 4, 'task_nodes': 1}}, 'conf_path': './examples/LDPDev/test_predict_conf.json', 'dsl_version': 2, 'role': {'host': [9998], 'guest': [9999]}, 'component_parameters': {'role': {'host': {'0': {'dataio_1': {'with_label': False}, 'dataio_0': {'with_label': False}, 'reader_1': {'table': {'name': 'breast_hetero_host', 'namespace': 'experiment'}}, 'reader_0': {'table': {'name': 'default_credit_hetero_host', 'namespace': 'experiment'}}}}, 'guest': {'0': {'dataio_1': {'with_label': True, 'output_format': 'dense'}, 'dataio_0': {'with_label': True, 'output_format': 'dense'}, 'reader_1': {'table': {'name': 'breast_hetero_guest', 'namespace': 'experiment'}}, 'reader_0': {'table': {'name': 'default_credit_hetero_guest', 'namespace': 'experiment'}}}}}, 'common': {'hetero_secure_boost_0': {'task_type': 'classification', 'objective_param': {'objective': 'cross_entropy'}, 'num_trees': 3, 'validation_freqs': 1, 'encrypt_param': {'method': 'iterativeAffine'}, 'tree_param': {'max_depth': 3}}, 'evaluation_0': {'eval_type': 'binary'}}}, 'local': {'role': 'host', 'party_id': 9998}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO', 'output_data_name': ['data']}
[INFO] [2021-06-15 21:31:55,403] [2573:139793912252224] - task_executor.py[line:152]: Task input dsl {'model': ['pipeline.dataio_0.model']}
[ERROR] [2021-06-15 21:31:55,429] [2573:139793912252224] - task_executor.py[line:191]: not enough values to unpack (expected 4, got 2)
Traceback (most recent call last):
  File "/fate/python/fate_flow/operation/task_executor.py", line 168, in run_task
    run_object.run(component_parameters_on_party, task_run_args)
  File "/fate/python/federatedml/model_base.py", line 87, in run
    running_funcs = self.component_properties.extract_running_rules(args, self)
  File "/fate/python/federatedml/util/component_properties.py", line 238, in extract_running_rules
    train_data, validate_data, test_data, data = self.extract_input_data(args, model)
ValueError: not enough values to unpack (expected 4, got 2)
[INFO] [2021-06-15 21:31:55,430] [2573:139793912252224] - task_executor.py[line:313]: report task 2021061521315031191914_dataio_0 0 host 9998 to driver
[INFO] [2021-06-15 21:31:55,498] [2573:139793912252224] - task_executor.py[line:202]: task 2021061521315031191914_dataio_0 host 9998 start time: 2021-06-15 21:31:54
[INFO] [2021-06-15 21:31:55,499] [2573:139793912252224] - task_executor.py[line:204]: task 2021061521315031191914_dataio_0 host 9998 end time: 2021-06-15 21:31:55
[INFO] [2021-06-15 21:31:55,499] [2573:139793912252224] - task_executor.py[line:206]: task 2021061521315031191914_dataio_0 host 9998 takes 0.636s
[INFO] [2021-06-15 21:31:55,499] [2573:139793912252224] - task_executor.py[line:209]: Finish 2021061521315031191914 dataio_0 2021061521315031191914_dataio_0 0 host 9998 task failed
[INFO] [2021-06-15 21:31:55,499] [2573:139793912252224] - task_executor.py[line:313]: report task 2021061521315031191914_dataio_0 0 host 9998 to driver
